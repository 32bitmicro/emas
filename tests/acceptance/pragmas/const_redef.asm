.const a 10
.const a 100
