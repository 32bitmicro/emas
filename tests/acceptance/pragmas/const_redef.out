acceptance/pragmas/const_redef.asm:2:13: Symbol 'a' already defined
